VideoApi/VideoApi.UnitTests/Controllers/Conference/BookNewConferenceTests.cs

```diff
@@ -236,5 +219,10 @@ public async Task Should_save_all_successfully()
 
             QueryHandlerMock.Verify(q => q.Handle<GetConferenceByIdQuery, VideoApi.Domain.Conference>(It.IsAny<GetConferenceByIdQuery>()), Times.Once);
         }
+        
+        protected virtual IEnumerable<T> GetEmptyArray<T>()
+        {
+            return Array.Empty<T>();
```

Comment:
`Array.Empty<T>()` is already verbose. I don't see any added value with this method

Reply:
this is a suggestion from sonar cloud

Reply:
I suspect sonar suggested to use `Array.Empty<T>();`. I don't believe we need a wrapper for a method when the body of the method is a single line

Reply:
Sonar is suggesting Array.Empty<T>()... then because it is in many places is suggesting to use a method to centralise the call

Reply:
I don't believe this method adds anything over directly using the implementation directly

Reply:
method removed